NURESTConnection.js

```diff
@@ -78,7 +78,7 @@ export default class NURESTConnection extends NUObject {
                     respHeaders[key] = value;
                 });
 
-                const result = ({ data, headers: respHeaders });
+                const result = ({ data, headers: respHeaders, responseStatusCode: response.status });
```

Comment:
please send the whole response back! I am not sure why in the first place this was not done this way!

Reply:
@natabal sorry, I somehow thought you had approved, will revert!